test/rng_testsuite/interface_tests/distributions_methods.pass.cpp

```diff
@@ -60,8 +60,40 @@ check_params(oneapi::dpl::normal_distribution<T>& distr)
     Element_type<T> stddev = static_cast<Element_type<T>>(1);
     return ((distr.mean() != mean) || (distr.stddev() != stddev) ||
             (distr.min() > -std::numeric_limits<Element_type<T>>::max()) ||
-            (distr.max() < std::numeric_limits<Element_type<T>>::max()) ||
-            (distr.param().first != mean) || (distr.param().second != stddev));
+            (distr.max() < std::numeric_limits<Element_type<T>>::max()) || (distr.param().first != mean) ||
+            (distr.param().second != stddev));
+}
+
+template <class T>
+std::int32_t
+check_params(oneapi::dpl::exponential_distribution<T>& distr)
+{
+    Element_type<T> lambda = static_cast<Element_type<T>>(1);

```

Comment:
`static_cast<Element_type<T>>(1)` -> `Element_type<T>(1)` ?
Here and below for other `static_cast`

Reply:
done